src/main/java/it/gov/pagopa/payhub/activities/dao/ClassifyDao.java

```diff
@@ -0,0 +1,17 @@
+package it.gov.pagopa.payhub.activities.dao;
+
+import it.gov.pagopa.payhub.activities.exception.ClearClassifyIufException;
+
+/**
+ * Data Access Object interface for saving payments classification
+ */
+public interface ClassifyDao {
+	/**
+	 * delete classification
+	 *
+	 * @param paymentReportingId reporting flow unique identifier
+	 * @param classification classification to delete
+	 */
+	void deleteClassificationByIuf(Long paymentReportingId, String classification) throws ClearClassifyIufException;
```

Comment:
this should just accept iuf, classification (and maybe the organizationId)
I don't think we should declare a custom exception as to be thrown

Reply:
OK